concepts/05 UI Components/Splitter/20 Implement Custom Scrolling/00 Implement Custom Scrolling.md

```diff
@@ -0,0 +1,222 @@
+Splitter displays native scroll bars when a pane content is smaller than the pane itself (for instance, as a result of resize).
```

Comment:
I guess it should be the other way - pane content should be **bigger** than the pane. Then you'll see scrollbars. 

```suggestion
Splitter displays native scroll bars when pane content doesn't fit into the pane. This often happens as a result of a resize operation.
```